docs/how-to/dnspolicy_basic.md

```diff
@@ -0,0 +1,52 @@
+# Defining a basic DNSPolicy
+
+## What is a DNSPolicy
+
+DNSPolicy is a Custom Resource Definition supported by the MGC that follows the
+[policy attachment model](https://gateway-api.sigs.k8s.io/references/policy-attachment/), which allows users to enable and configure DNS against the Gateway.
+
+This document describes how to enable DNS by creating a basic DNSPolicy
+
+## Pre-requisites
+
+* A [ManagedZone](managedZone.md) has been created
+* A Gateway has been created with an attached HTTPRoute
```

Comment:
maybe we should call out that the DNSPolicy can be created before the HTTPRoute but that the record will only be created once a route is attached to the gateway?